IFC4x3/Properties/r/ReferenceEnvironmentTemperature_2vgOshA4D0$ems5HSBwemr/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="ReferenceEnvironmentTemperature_2vgOshA4D0ems5HSBwemr" Name="ReferenceEnvironmentTemperature" UniqueId="b9a98dab-2843-40fe-8c36-15170bea8c35" PropertyType="p_boundedvalue" PrimaryDataType="IfcRatioMeasure" />
```

Comment:
Datatype should be a thermal measure?

Reply:
Changed to existing "DesignAmbientTemperature" commit : 9649afe4f533232b66830889ebefe20fc9e51a00